lib/linguist/languages.yml

```diff
@@ -1411,9 +1411,14 @@ Groff:
   - .n
   - .rno
   - .roff
-  tm_scope: text.groff
+  - .tmac
+  filenames:
+  - mmn
```

Comment:
Can it also be `.mmn`? In your search results for that filename there seem to be quite a few `.mmn` files...